### PR TITLE
Remove N-Parsons from maintainers list

### PR DESCRIPTION
**Pull request by N-Parsons**
Unfortunately I haven't had the time/energy to keep up work as a maintainer, so I'm removing myself from the maintainers list.

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -20,16 +20,6 @@
       "link_url": null,
       "avatar_url": null
     },
-    {
```
Comment (cmccandless) [inline config/maintainers.json]: If you would like to remain marked as an alumnus, you'll need to add the following in place of your old entry:
```
{
  "github_username": "N-Parsons",
  "alumnus": true
},
```

**Comment by cmccandless** [comment]
cc @kytrinyx

**Comment by kytrinyx** [comment]
Once this gets rebased (to fix the conflict) and merged, I'll double-check the team memberships to make sure the bot got the job done.

**Comment by Dog** [comment]
I went ahead and handled the conflict as given. @N-Parsons if you'd like to be changed to alumnus see @cmccandless 's comment.

**Comment by Dog** [comment]
Since it has been a few days I'm going to go ahead and merge this. If you'd like to update it, then please submit another PR.

**Pull request merged**